src/library/scala/collection/immutable/Stream.scala

```diff
@@ -301,11 +301,11 @@ sealed abstract class Stream[+A] extends AbstractSeq[A] with LinearSeq[A] with L
     *
     *         Examples:
     *
-    *           - `"LazyList(_, ?)"`, a non-empty lazy list, whose head has not been
+    *           - `"Stream(_, ?)"`, a non-empty stream, whose head has not been
     *             evaluated ;
-    *           - `"LazyList(_, 1, _, ?)"`, a lazy list with at least three elements,
+    *           - `"Stream(_, 1, _, ?)"`, a stream with at least three elements,
```

Comment:
This could use some fixing in that `Stream` is not lazy in the head, so it's not possible for the head not to be evaluated, but it's still strictly better than before